app/javascript/components/cloud-volume-form/cloud-volume-form.schema.js

```diff
@@ -12,12 +12,48 @@ const changeValue = (value, loadSchema, emptySchema) => {
 
 const storageManagers = (supports) => API.get(`/api/providers?expand=resources&attributes=id,name,${supports}&filter[]=${supports}=true`)
   .then(({ resources }) => {
-    let storageManagersOptions = [];
-    storageManagersOptions = resources.map(({ id, name }) => ({ label: name, value: id }));
+    let storageManagersOptions = resources.map(({ id, name }) => ({ label: name, value: id }));
     storageManagersOptions.unshift({ label: `<${__('Choose')}>`, value: '-1' });
     return storageManagersOptions;
   });
 
+// storage manager functions to filter by capabilities:
+const equalsUnsorted = (arr1, arr2) => {
+  if (arr1.length !== arr2.length) return false;
+  const uniqueValues = new Set([...arr1, ...arr2]);
+  for (const v of uniqueValues) {
+    const aCount = arr1.filter(e => e === v).length;
+    const bCount = arr2.filter(e => e === v).length;
+    if (aCount !== bCount) return false;
+  }
+  return true;
+}
+
+const filterServicesByCapabilities = (filterArray) => API.get(`/api/storage_services?expand=resources&attributes=id,name,capabilities`)
```

Comment:
`filterServicesByCapabilities` looks similar to `filterResourcesByCapabilities`

maybe we can merge these 2 methods to one by passing additional parameters (only if possible, it works wither way!)